### PR TITLE
[UCDC] Add additional fields form

### PR DESCRIPTION
**Pull request by arsentur**
**Description:** 
- Add Dual filed for user profession
- Expand `UserProfile` model with additional fields
- Add modal window to the progress page.
- Add to the progress page `UserAccount` form with needed fields
- Create js handler for managing new fields and forms.

**Youtrack:** 
[UCDC-78](https://youtrack.raccoongang.com/issue/UCDC-78)

### CODE REVIEW

File: lms/envs/common.py [hunk 1]
```diff
@@ -3063,6 +3063,14 @@ def _make_locale_paths(settings):
         "account_privacy",
         "accomplishments_shared",
         "extended_profile",
+        "position",
+        'first_name',
```
Comment (idegtiarov) [inline lms/envs/common.py]: Please pay attention to the quotation it is a good practice to use similar quotes/double quotes in the same block.

File: common/djangoapps/student/models.py [hunk 2]
```diff
@@ -383,7 +399,7 @@ class Meta(object):
         ('m', ugettext_noop('Male')),
         ('f', ugettext_noop('Female')),
         # Translators: 'Other' refers to the student's gender
-        ('o', ugettext_noop('Other/Prefer Not to Say'))
+        # ('o', ugettext_noop('Other/Prefer Not to Say'))
```
[the remark this answers is not in the document]
Comment (arsentur) [reply common/djangoapps/student/models.py]: Removed unnecessary option. Done

File: lms/djangoapps/courseware/views/views.py [hunk 3]
```diff
@@ -104,6 +104,7 @@
 from xmodule.tabs import CourseTabList
 from xmodule.x_module import STUDENT_VIEW
 
+from student_account.views import account_settings_context
```
[the remark this answers is not in the document]
Comment (arsentur) [reply lms/djangoapps/courseware/views/views.py]: Fixed. Done

File: lms/djangoapps/courseware/views/views.py [hunk 4]
```diff
@@ -980,6 +981,17 @@ def _progress(request, course_key, student_id):
     # checking certificate generation configuration
     enrollment_mode, _ = CourseEnrollment.enrollment_mode_for_user(student, course_key)
 
+    profile = student.profile
+
+    can_request_certificate = all([
```
[the remark this answers is not in the document]
Comment (arsentur) [reply lms/djangoapps/courseware/views/views.py]: Added comment

File: lms/envs/common.py (hunk 1 above)
Comment (arsentur) [reply lms/envs/common.py]: Fixed

File: lms/static/js/views/fields.js [hunk 5]
```diff
@@ -693,6 +695,81 @@
...
+
+            fieldTemplate: field_dual_template,
+
+            events: {
+                'change input': 'saveFieldsValues',
+                'change select': 'saveFieldsValues'
+            },
+
+
+
```
[the remark this answers is not in the document]
Comment (arsentur) [reply lms/static/js/views/fields.js]: Done

**Pull request merged**